### ISSUE TITLE
[Android] Even though dev-options have been changed, scores are still getting saved

### ISSUE THREAD

**Issue by Sgt-Miller** [agateau/pixelwheels#159]
Steps to recreate
Go to Settings -> Under the hood -> Dev options
Switch on "One lap only"
Message appears saying "settings affecting game play have been modified, scores won't be saved"

Lets say for a map my record previously was :
Best Lap : 10s
Best Total : 30s

Now play map, game completes after one round. Lets say it took 12s to complete. 

Now the record is shown as
Best Lap : 10s
Best Total : 12s (even though I only completed one lap). 

Basically, eventhough warning "settings affecting game play have been modified, scores won't be saved" , scores are still getting saved. 

PS : Fantastic game, have been playing for so many hours now

**Comment by MinekPo1** [comment]
I sadly also encountered this issue.

**Comment by agateau** [comment]
Thanks for your report, looking into it.

**Comment by agateau** [comment]
Which version is this happening on? The bug should be fixed in 0.21.0.

**Comment by Sgt-Miller** [comment]
Hi, I tried it two times. In first case, score was still there after restarting (and is still there). In second case, after restart the score was removed. 
I'll try again tomorrow with other options too.

**Comment by Sgt-Miller** [comment]
No longer able to recreate this issue!

**Comment by agateau** [comment]
> No longer able to recreate this issue! Not sure how it happened that time.

Thanks for taking the time to check :+1: .
